### DISCUSSION TITLE
I can not connect the MAC plugin to the server WIN

### DISCUSSION THREAD

**Discussion by Guswis** [apohl79/audiogridder#1382] [Help]
Hello
I recently connected a PC with Win 11 home as a server and I was unable to establish a connection to the MAC OSX CATALINA, I disabled the firewall etc. I gave permissions to the AG and I was not successful in connecting to the plugin. Although the machines are on the network but not AG.
I'm using between macs server and plugin with no problem.
Can someone help me?

**Comment by iamhosni74** [comment]
Hello Guswis,
Here also,  I’ve gone through all the steps trying to connect my Mac to the PC (server), but I’m still not having any luck.
 If anyone has any tips or guidance, I would really appreciate your help. Thanks a lot!

**Reply by abab6111** [reply]
Can you use terminal ping from mac to win?
